wp-content/themes/core/components/container/container.php

```diff
@@ -0,0 +1,13 @@
+<?php
+declare( strict_types=1 );
+
+/**
+ * @var array $args Arguments passed to the template
+ */
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$controller = \Tribe\Project\Templates\Components\container\Controller::factory( $args );
+
+?>
+<<?php echo $controller->tag(); ?> <?php echo $controller->classes(); ?> <?php echo $controller->attributes(); ?>>
```

Comment:
Formatting. I've noticed we often break this calls to multiple lines for readability. @ryanurban can confirm.

Reply:
Yeah, I can do a pass on this later, post merge, it's a react thing : )